django/contrib/admin/static/admin/js/actions.js

```diff
@@ -109,7 +109,7 @@
             lastChecked = target;
             updateCounter();
         });
-        $('form#changelist-form table#result_list tr').find('td:gt(0) :input').change(function() {
+        $('form#changelist-form table#result_list tr').on('change', 'td:nth-child(n+2) :input', function() {
```

Comment:
Maybe the significance of n+2 merits a comment? (In fact, I'm not sure if it's correct -- for example, is it correct if the checkbox column is removed with `admin.site.disable_action('delete_selected')`?)